Lib/test/test_urllib.py

```diff
@@ -1531,10 +1532,12 @@ def test_url2pathname_win(self):
     def test_url2pathname_posix(self):
         fn = urllib.request.url2pathname
         self.assertEqual(fn('/foo/bar'), '/foo/bar')
-        self.assertEqual(fn('//foo/bar'), '//foo/bar')
+        self.assertRaises(urllib.error.URLError, fn, '//foo/bar')
         self.assertEqual(fn('///foo/bar'), '/foo/bar')
```

Comment:
Some tests may work on Posix and on Windows. It is better to move them into common test method.

Are there tests for relative path, not starting with a slash?

Reply:
:+1: done

There were already a couple of test cases for relative paths/URLs in `test_basic` and `test_quoting`, FWIW.